### DISCUSSION TITLE
Ability to set Boolean column true/false values

### DISCUSSION THREAD

**Discussion by gavinhewitt** [filamentphp/filament#304] [Ideas]
In my application I have a column 'status' that's either active or inactive. Nova allows you to set the trueValue of falseValue. Would be awesome if Filament had similar functionality.

**Comment by ryangjchandler** [comment]
You mean the `Checkbox` field?

**Reply by gavinhewitt** [reply]
No sorry, I meant the Boolean column. Changed title accordingly.

**Reply by ryangjchandler** [reply]
You can use a `View` column and output the value however you want. The `Boolean` column is just designed for use with actual booleans right now, hence the tick/cross icons. It also extends the `Icon` column so adding in support for `trueValue` and `falseValue` would add an extra level of complexity that we probably don't need right now, especially when a user land implementation exists.

**Reply by gavinhewitt** [reply]
OK, thanks
